### PR TITLE
Infinity treatment

### PR DESCRIPTION
**Pull request by cdoehn**
## Description.

Some measurement values from the laser scanner have special meanings:

59956: no reflective signal arrived,
59958: a reflective signal arrived, but it was too late (according to our time window)

both should be published as +infinity

Before the PR the measurments are published as described above. And after the PR +infinity values are published instead of 59956 or 59958.

### Things to add, update or check by the maintainers before this PR can be merged.

- [x] ~Public api function documentation~
- [x] ~Architecture documentation reflects actual code structure~
- [x] ~[Tutorials](https://wiki.ros.org/pilz_robots/Tutorials/)~
- [x] ~Overview on [ROS wiki](https://wiki.ros.org/pilz_robots)~
- [x] Package Readme ([example pilz_robots](https://github.com/PilzDE/pilz_robots/blob/melodic-devel/README.md))
- [x] ~Good commit messages ([some tips](https://dev.to/jacobherrington/how-to-write-useful-commit-messages-my-commit-message-template-20n9))~
- [x] CHANGELOG.rst updated
- [x] ~Copyright headers~
- [x] ~Examples~

### Review Checklist
- [x] ~Soft- and hardware architecture (diagrams and description)~
- [x] Test review (test plan and individual test cases)
- [x] ~Documentation describes purpose of file(s) and responsibilities~
- [x] Code (coding rules, style guide)

### Release planning (please answer)
- [x] ~When is the new feature released?~
- [x] ~Which dependent packages do have to be released simultaneously?~

### CODE REVIEW

File: standalone/src/data_conversion_layer/monitoring_frame_deserialization.cpp [hunk 1]
```diff
@@ -53,6 +53,10 @@ FixedFields::FixedFields(DeviceStatus device_status,
 
 static constexpr double toMeter(const uint16_t& value)
 {
+  if ((value == NO_SIGNAL_ARRIVED) || (value == SIGNAL_TOO_LATE))
+  {
+    return INFINITY;
```
Comment (agutenkunst) [inline standalone/src/data_conversion_layer/monitoring_frame_deserialization.cpp:58]: How does this relate to https://en.cppreference.com/w/cpp/types/numeric_limits/infinity

**Comment by ct2034** [comment]
Could you please clarify in the description:

- What issue is the PR adressing?
- What was the behavious before the PR, what is it after?
- Is there an issue for that?
- Can you please clean up the to do list in the description.

**Review by ct2034 — CHANGES_REQUESTED** [review]
Could you please clarify in the description:

- What issue is the PR adressing?
- What was the behavious before the PR, what is it after?
- Is there an issue for that?
- Can you please clean up the to do list in the description.

File: standalone/src/data_conversion_layer/monitoring_frame_deserialization.cpp (hunk 1 above)
Comment (cdoehn) [reply standalone/src/data_conversion_layer/monitoring_frame_deserialization.cpp]: I changed it accordingly

**Comment by cdoehn** [comment]
> * Is there an issue for that?

No

**Review by agutenkunst — CHANGES_REQUESTED** [review]
Everything fine!

Just update the Changelog and I will merge!

**Pull request merged**